### DISCUSSION TITLE
Little question

### DISCUSSION THREAD

**Discussion by RedCarlos26** [AntiCope/meteor-rejects#76] [bruh]
Is the addon compatible with ion?

**Comment by C10udburst** [comment]
i have no idea, but there's nothing stopping you from trying

**Reply by RedCarlos26** [reply]
Alright I'm gonna try

**Comment by sbxte** [comment]
How'd it go? Is it compatible?

**Reply by RedCarlos26** [reply]
Didn't tried yet, I will do that tonight

**Reply by RedCarlos26** [reply]
Alright. I tried as well just forgot to mention it here lol

**Comment by ghost** [comment]
how do i use force opand what does it do

**Reply by C10udburst** [reply]
wtf is force opand
